### DISCUSSION TITLE
UnRaid docker issues

### DISCUSSION THREAD

**Discussion by DrewBiermann** [JonathanTreffler/backblaze-personal-wine-container#163] [Q&A]
I'm having some issues getting the UnRaid docker to work. I can get through the install of backblaze in WINE ok. However, it's mounting the system as Z: for some reason and not D:. When I attempt to add the Z: drive as a backup it says it can't create the .bzvol directory on the hard drive.

**Comment by traktuner** [comment]
Hey @DrewBiermann 
Can you please share your startapp.log file?
`/config/wine/dosdevices/c:/backblaze-wine-startapp.log`
Thank you!

**Reply by CreakyLid** [reply]
Just popping in to say I am also having this issue on Unraid! Attaching my backblaze-wine-startapp.log file if it is of any help.

[Here are some screenshots of the VNC window.](https://imgur.com/a/tPQWisn)

In the VNC window, I have the Backblaze control panel, which is currently backing up "C:\" that is ~30 MB, and the only other drive option is "Z:\", which when selected gives the '.bzvol' error.

I also get a popup about permission access to to the 'bzreports' directory too? You can see this in the screenshots above.

Anything else helpful just lemme know!
[backblaze-wine-startapp.log](https://github.com/user-attachments/files/15946121/backblaze-wine-startapp.log)

**Comment by avarage-hero** [comment]
i have the same problem and when i try the .log is says  Permission denied
